code/__DEFINES/atmospherics/atmos_mob_interaction.dm

```diff
@@ -25,6 +25,8 @@
 #define MIASMA_CORPSE_MOLES 0.02
 #define MIASMA_GIBS_MOLES 0.005
 
+/// Minimum amount of toxic gas moles in a single breath required to receive damage
+#define TOXIC_GAS_DAMAGE_MIN_MOLES (MOLES_GAS_VISIBLE * BREATH_PERCENTAGE)
```

Comment:
this feels like a weird define if its only for tritium?

Reply:
Right now I added it just for tritium, but it can be reused for something else, since @Iamgoofball mentioned that this is a rule.

Reply:
Other gases use partial pressure (percentage of gas in the breathed mixture). While the rule is to deal damage when the gas is visible, which is based on moles, not percentage. I'm unaware of reasons for this, that's why I didn't touch other checks. Tritium one looked like a bug, because it worked even when the partial pressure is 0.

Reply:
if its not reused right now, it feels misleading